src/Setup.php

```diff
@@ -415,5 +416,88 @@ public function bcgov_block_theme_wp_headers( $headers ) {
+			'featured_image'           => esc_html__( 'Featured image for this Custom Pattern', 'bcgov-block-theme' ),
+			'set_featured_image'       => esc_html__( 'Set featured image for this Custom Pattern', 'bcgov-block-theme' ),
+			'remove_featured_image'    => esc_html__( 'Remove featured image for this Custom Pattern', 'bcgov-block-theme' ),
+			'use_featured_image'       => esc_html__( 'Use as featured image for this Custom Pattern', 'bcgov-block-theme' ),
+			'archives'                 => esc_html__( 'Custom Pattern archives', 'bcgov-block-theme' ),
+			'insert_into_item'         => esc_html__( 'Insert into Custom Pattern', 'bcgov-block-theme' ),
+			'uploaded_to_this_item'    => esc_html__( 'Upload to this Custom Pattern', 'bcgov-block-theme' ),
+			'filter_items_list'        => esc_html__( 'Filter Custom Patterns list', 'bcgov-block-theme' ),
+			'items_list_navigation'    => esc_html__( 'Custom Patterns list navigation', 'bcgov-block-theme' ),
+			'items_list'               => esc_html__( 'Custom Patterns list', 'bcgov-block-theme' ),
+			'attributes'               => esc_html__( 'Custom Patterns attributes', 'bcgov-block-theme' ),
+			'name_admin_bar'           => esc_html__( 'Custom Pattern', 'bcgov-block-theme' ),
+			'item_published'           => esc_html__( 'Custom Pattern published', 'bcgov-block-theme' ),
+			'item_published_privately' => esc_html__( 'Custom Pattern published privately.', 'bcgov-block-theme' ),
+			'item_reverted_to_draft'   => esc_html__( 'Custom Pattern reverted to draft.', 'bcgov-block-theme' ),
+			'item_scheduled'           => esc_html__( 'Custom Pattern scheduled', 'bcgov-block-theme' ),
+			'item_updated'             => esc_html__( 'Custom Pattern updated.', 'bcgov-block-theme' ),
+			'parent_item_colon'        => esc_html__( 'Parent Custom Pattern:', 'bcgov-block-theme' ),
+		];
+
+		$args = [
+			'label'                 => esc_html__( 'Custom Patterns', 'bcgov-block-theme' ),
+			'labels'                => $labels,
+			'description'           => '',
+			'public'                => true,
+			'publicly_queryable'    => true,
+			'show_ui'               => true,
+			'show_in_rest'          => true,
+			'rest_base'             => '',
+			'rest_controller_class' => 'WP_REST_Posts_Controller',
+			'rest_namespace'        => 'wp/v2',
+			'has_archive'           => false,
+			'show_in_menu'          => true,
+			'show_in_nav_menus'     => true,
+			'delete_with_user'      => false,
+			'exclude_from_search'   => false,
+			'capability_type'       => 'post',
+			'map_meta_cap'          => true,
+			'hierarchical'          => false,
+			'can_export'            => false,
+			'rewrite'               => [
+				'slug'       => 'custom-pattern',
+				'with_front' => true,
+			],
+			'query_var'             => true,
+			'menu_position'         => 20,
+			'menu_icon'             => 'dashicons-editor-kitchensink',
+			'supports'              => [ 'title', 'editor', 'revisions' ],
+			'taxonomies'            => [ 'category' ],
```

Comment:
i didn't know you could do this with taxonomies, however does this share the same one with the `post` post type.  If it shares it, not sure we want to do this.

Reply:
Changing this to a custom taxonomy / category for Patterns.